lib/chef_zero/endpoints/organization_user_endpoint.rb

```diff
@@ -20,29 +20,7 @@ def delete(request)
         json_response(200, ChefData::DataNormalizer.normalize_user(user, request.rest_path[3], ['username'], server.options[:osc_compat]))
       end
 
-      def post(request)
```

Comment:
I've moved this in https://github.com/andrewjamesbrown/chef-zero/commit/04470183de1bb51edeb29f7894bb965306b7faeb which is part of https://github.com/chef/chef-zero/pull/146